### PR TITLE
Add passive event support for keen-slider

### PR DESCRIPTION
**Pull request by boris-felix**
## Summary of Change

As per the suggestion made here : https://github.com/loveholidays/sunrise/pull/821
We want to maintain the lighhouse score on Sunrise by supporting passive event into keen-slider.
To be able to identify easily the piece of changes, i simply forked the project and did the change here to then potentially push it back to the main repository.

Added a pollyfill to support passive events on legacy browsers.

### CODE REVIEW

File: src/keen-slider.js [hunk 1]
```diff
@@ -50,8 +50,22 @@ function KeenSlider(initialContainer, initialOptions = {}) {
...
+    let supportsPassive = false;
+    try {
+      let opts = Object.defineProperty({}, 'passive', {
+        get: function() {
+          supportsPassive = true;
+        }
+      });
+      window.addEventListener("testPassive", null, opts);
+      window.removeEventListener("testPassive", null, opts);
+    } catch (e) {}
```
Comment (dios-david) [inline src/keen-slider.js]: you could evaluate this once and store it somewhere. now this is being evaluated for each `eventAdd` calls

File: src/polyfills.js [hunk 2]
```diff
@@ -3,3 +3,227 @@ if (!Math.sign) {
     return (x > 0) - (x < 0) || +x
   }
 }
+
+// From : https://github.com/WebReflection/dom4/blob/master/src/event-target.js
+(function (global){
```
Comment (dios-david) [inline src/polyfills.js]: why do we need this polyfill here?

Comment (boris-felix) [reply src/polyfills.js]: To support browsers which do not support passive events handling

**Comment by boris-felix** [comment]
The fix seems to work : https://lighthouse.lvh.systems/app/projects/sunrise/compare/aae140ef-bc7c-421c-a23c-959198efff01.
However my PR on `keen-slider` root project got closed as it would apparently negatively impact mobile experience ( that's what's keen-slider's author said ) and a patch has finally been released into `v5.2.2` 🎉 : https://github.com/rcbyr/keen-slider/pull/40. 
Closing the PR and upgrading the version on Sunrise.

**Pull request closed without merging**